### PR TITLE
Add development instructions

### PR DESCRIPTION
**Pull request by TimeTerminal**
Addresses #42

### CODE REVIEW

**Comment by TimeTerminal** [comment]
Ready for your review @btzr-io

File: examples/index.js [hunk 1]
```diff
@@ -11,7 +11,7 @@ const villainOpts = {
 }
 
 const testFile = '/build/test_files/archive.cbz'
```
Comment (btzr-io) [inline examples/index.js]: We don't need this anymore

File: examples/index.js [hunk 2]
```diff
@@ -11,7 +11,7 @@ const villainOpts = {
...
-const testFile2 = '/build/test_files/test.cbz'
+const testFile2 = '/build/testFiles/Example-archive.zip'
```
Comment (btzr-io) [inline examples/index.js]: rename to testFile

File: README.md [hunk 3]
```diff
@@ -26,20 +26,28 @@ An open source web-based comic book reader.
 $ yarn add villain-react
 ```
 
+## Development
+
+- Run `yarn install` in the console
```
Comment (btzr-io) [inline README.md]: I think should be just `yarn` not sure if there is any significant difference but is more simple :smile:

**Comment by btzr-io** [comment]
Nice, just some minor changes requested.

**Pull request merged**